sklearn/svm/src/liblinear/liblinear_helper.c

```diff
@@ -20,20 +20,23 @@ static struct feature_node **dense_to_sparse(double *x, npy_intp *dims,
 {
```

Comment:
The function docstring above would also need to be updated I think,
> We create a temporary array temp that collects non-zero
> elements and after we just memcpy that to the proper array.

Reply:
Good catch! I just force-pushed a new commit with a corrected docstring.